### PR TITLE
chore: add kr to codeowners

### PR DESCRIPTION
**Pull request by seaerchin**
## Problem
now kr is on our team, shouldn't have to manually add him to review

Closes [insert issue #]

## Solution
add him to codeowners file

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1 +1,2 @@
 * @opengovsg/isomer-engineers
+* @karrui
```
Comment (harishv7) [inline .github/CODEOWNERS:2]: why not add within the isomer-engineers team?

Comment (seaerchin) [reply .github/CODEOWNERS:2]: @karrui don't want

Comment (harishv7) [reply .github/CODEOWNERS:2]: hmm because of?

Comment (harishv7) [reply .github/CODEOWNERS:2]: usually we do add/remove within the team. not sure if there's any other reason

Comment (karrui) [reply .github/CODEOWNERS:2]: oh i didn't know so troublesome, just do it then LOL

Comment (seaerchin) [reply .github/CODEOWNERS:2]: ok i'll add you to the team and close this PR

**Pull request closed without merging**